packages/ladle/lib/cli/story-watcher.js

```diff
@@ -0,0 +1,48 @@
+import chokidar from "chokidar";
+
+/**
+ * Extract the base directory from a glob pattern.
+ * e.g., "src/**\/*.stories.tsx" -> "src"
+ * @param {string} pattern
+ * @returns {string}
+ */
+export const getGlobBasePath = (pattern) => {
+  // Normalise path separators to forward slashes for cross-platform compatibility
+  const normalised = pattern.replace(/\\/g, "/");
+  const parts = normalised.split("/");
+  const baseParts = [];
+  for (const part of parts) {
+    if (part.includes("*") || part.includes("{") || part.includes("[")) break;
+    baseParts.push(part);
+  }
+  return baseParts.length > 0 ? baseParts.join("/") : ".";
+};
+
+// Story file pattern - matches .stories.{js,jsx,ts,tsx,mdx}
+export const STORY_FILE_REGEX = /\.stories\.(js|jsx|ts|tsx|mdx)$/;
```

Comment:
we should not redefine this here, there is already a default and users can customize it

Reply:
Ah, apologies, missed that 😅 just pushed an update to use the given pattern and also realised picomatch can find the base path so removed the custom script to do it